

# Mask Contour:
float32 x
float32 y


